launch_testing/launch_testing/__init__.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         self.name = name
         self.launch_descriptor = launch_descriptor
         self.expected_lines = expected_lines
+        self.expected_output = b'\n'.join(self.expected_lines)
```

Comment:
I mean a single regex which can match across newlines. I thought this PR only aims to support multiple single-line regexps?